### ISSUE TITLE
Ability to set default settings for a pad applied to all users

### ISSUE THREAD

**Issue by nclm** [ether/etherpad-lite#6034]
* * *

name: Feature request
about: Suggest an idea for this project
title: ''
labels: enhancement
assignees: 

* * *

**Is your feature request related to a problem? Please describe.**
Depending on the content of a pad and its audience, some settings make more sense or not (language, font choice, line numbers or not, authorship colours or not, etc). Of course individual users might have personal preferences, but the pad creator or members might also have a good idea of what is needed for this pad. Currently (from experience using board.net mostly, apologies if this isn’t correct), there are personal settings that are saved locally for each user, but when sharing the pad to other users, they get the default settings.

**Describe the solution you'd like**
A way for settings to be saved as part of the pad itself, so that when shared, the settings stay applied. Of course users would still be able to overwrite that and apply their own settings, but the default for each specific pad would not necessarily be the default of Etherpad/the provider. In its simplest form, it would be a “Apply settings as default for this pad” button in the settings pop-over. Currently, there the settings are labelled “Pad Settings” and “My View”, and that can be rethought as well, to make it clear that there are default settings for the pad, and personal settings changing that.

**Describe alternatives you've considered**
There are many ways to do this UI wise, the simplest being the addition of one button described above. I don’t see other solutions for sharing settings across users of a pad than adding this feature.

**Additional context**
Technically, it also mean storing the settings somewhere on server.

**Comment by SamTV12345** [comment]
Thanks for the issue. Maybe a plugin could  solve that usecase.

**Comment by JulesLecuir** [comment]
Hi, I agree. That should be a default setting that the pad owner can modify, and that pad readers can override if they want to. Not sure if a plugin should manage this. For me this is such a basic feature that etherpad should manage on its own.

The most painful is that table of contents is displayed by default, which results in a terrible user experience because all the text is compressed on the right :
![image](https://github.com/ether/etherpad-lite/assets/33055419/ae092a93-7329-4f92-a854-2ac485a9df3b)

**Comment by nclm** [comment]
Yes I also agree that this sounds more within the scope of the default app than an optional plug-in. The current behaviour is unexpected and the experience would be improved if that was a default option.

**Comment by JohnMcLear** [comment]
This goes in with "Delete pad" where we should have a "Creator" pad view under Settings and that Creator applies default settings.  Essentially the "initial author" would get these "rights" and no one else would see them..  I guess we might need a way for the initial author to make other authors "pad admins" too but that might be a v2 thing?

ref https://github.com/ether/etherpad/issues/7528
